.github/workflows/pr_automations.yml

```diff
@@ -71,4 +71,4 @@ jobs:
           github-token: ${{ secrets.ACCESS_TOKEN }}
           script: |
             const { main } = await import('${{ github.workspace }}/automations/js/src/project_automation/prs.mjs')
-            await main(github)
+            await main(github, core)
```

Comment:
We need to pass `core` from the YAML files to the `main()` functions in `issues.mjs` and `prs.mjs`.